web/src/pages/document/document-page.tsx

```diff
@@ -70,97 +68,42 @@ export function DocumentPage({
-        setGoToPage(page);
-    };
-
-    const handleGoNext = useCallback(() => {
-        isLastPage ? setGoToPage(pages) : setGoToPage((prev) => prev + 1);
-    }, [goToPage, pages, isLastPage]);
-
-    const handleGoPrev = useCallback(() => {
-        isFirstPage ? setGoToPage(1) : setGoToPage((prev) => prev - 1);
-    }, [goToPage, pages, isFirstPage]);
-
-    const pagesDataSource = useArrayDataSource(
-        {
-            items: Object.values(getPageNumbers(pages!)),
-            getId: (item) => item
-        },
-        []
-    );
-
     crumbs.push({ name: fileMetaInfo.name });
 
+    const onCurrentPageChange = useCallback((pageOrderNumber: number) => {
+        documentPDFRef.current?.scrollDocumentTo(pageOrderNumber);
+    }, []);
+
     return (
-        <div className={styles['document-page']}>
-            <div className={styles.header}>
-                <div className={styles['header__left-block']}>
-                    <BreadcrumbNavigation breadcrumbs={crumbs} />
-                    <FlexRow>
-                        <FlexRow cx={styles['goto-page-selector']}>
-                            <FlexCell minWidth={60}>
-                                <span>Go to page</span>
-                            </FlexCell>
-                            <PickerInput
-                                minBodyWidth={52}
-                                size="24"
-                                dataSource={pagesDataSource}
-                                value={goToPage}
-                                onValueChange={onPageChange}
-                                getName={(item) => String(item)}
-                                selectionMode="single"
-                                disableClear={true}
-                            />
-                            <FlexRow>
-                                <span>of {pages}</span>
-                                <Button
-                                    size="24"
-                                    fill="white"
-                                    icon={goPrevIcon}
-                                    cx={styles.button}
-                                    onClick={handleGoPrev}
-                                    isDisabled={isFirstPage}
-                                />
-                                <Button
-                                    size="24"
-                                    fill="white"
-                                    icon={goNextIcon}
-                                    cx={styles.button}
-                                    onClick={handleGoNext}
-                                    isDisabled={isLastPage}
-                                />
-                            </FlexRow>
+        <TaskAnnotatorContextProvider
+            jobId={documentJobId}
+            revisionId={documentJobRevisionsInfo?.selectedDocumentJobRevisionId}
+            fileMetaInfo={fileMetaInfo}
+            onRedirectAfterFinish={() => {}}
```

Comment:
For convention reasons, what do you think about using that `noop` function instead of this empty callback?

Reply:
This logic was not introduced by me, this was just move from another place. I think "noop" is actually not a convention for the project, this was just someone's wish + we have a task to remove "lodash" from the project at all, so, I think it's okay for now to leave it as is.